### PR TITLE
add specific test for includepad

### PR DESCRIPTION
**Pull request by hughperkins**

### CODE REVIEW

File: test.lua [hunk 1]
```diff
@@ -1709,6 +1709,30 @@ function cunntest.SpatialMaxPooling_backward_batch_atomic()
    mytester:assertlt(error:abs():max(), precision_backward, 'error on state (backward) ')
 end
 
+function cunntest.SpatialAveragePooling_includepad()
+   local rngState = torch.getRNGState()
```
Comment (fmassa) [inline test.lua]: There is no randomness in this test, should we just take off those RGNs ?

**Comment by fmassa** [comment]
I have just a small remark, except from that it's good to go.

**Comment by hughperkins** [comment]
Fair point :-)  Wil lremove the rngs.

**Comment by hughperkins** [comment]
(compiling :-( )

**Comment by hughperkins** [comment]
Done.  Example output:

```
user@pear:~/git/cunn$ luajit -l cunn -e 'nn.testcuda({"SpatialAveragePooling_includepad"})'
seed:   1443867643
Running 1 tests
_  ==> Done                            

Completed 3 asserts in 1 tests with 0 errors
```

**Comment by fmassa** [comment]
great ! thanks a lot

**Pull request merged**